src/app/[locale]/specs/lexicon/en.mdx

```diff
@@ -208,6 +231,34 @@ Note that unlike `object`, there is no `nullable` field on `params`.
 
 `params` can not be declared as a top-level named type in a schema `defs` array. This means that they can not be referenced by a `ref` or `union`.
 
+### `permission`
+
+This is a limited-scope type which is only ever used for the `permissions` field of the `permission-set` primary type, and should not appear in other schema contexts. They map to auth permissions.
+
+Type-specific fields:
+
+- `resource` (string, required): indicates what account resource is being accessed
+
+Additional `permission` fields depend on the resource type being declared. The set of resources may expand over time, but currently includes:
+
+- `repo`: public repository write permissions
+    - `collection` (array of strings, required): indicates record types. Must include at least one element (not empty array). Supports wildcard (`*`), but does not support partial-NSID glob patterns (`com.example.*`)
+    - `action` (array of strings, optional): limits set of record operations. Values must be one or `create`, `update`, `delete`, case-sensitive.
+- `rpc`: remote API calls via proxying or service token generation
+    - `lxm` (array of strings, required): allowed XRPC endpoints, as NSIDs. Must include at least one element (not empty array). Supports wildcard (`*`), but does not support partial-NSID glob patterns (`com.example.*`)
+    - `aud` (string, semi-required): remote service, identified by DID service reference (DID followed by required service type fragment, eg `did:web:api.example.com#srvtype`). Supports wildcard (`*`). Required unless `inheritAud` is true.
+    - `inheritAud` (boolean, optional): if true, `aud` does not need to be defined.
+- `blob`: media file upload permissions
+    - `accept` (array of strings, required): mimetypes or partial mimetype glob patterns (`*/*` or `text/*` for example). Same syntax as the `accept` field in the `blob` lexicon type.
+- `account`: hosted account management
+    - `attr` (string, required): an aspect or component of account. Wildcard is not supported.
+    - `action` (string, optional): degree of control. Currently supports `read` or `manage`
+- `identity`: network identity (DID) management
+    - `attr` (string, required): an aspect or component of account. May be be wildcard (`*`)
+    - `action` (string, optional): degree of control. Currently supports `manage` or `submit`
```

Comment:
There are not allowed in permission sets.

```suggestion
`account` and `identity` permissions **cannot** be granted through permission sets and **must** be ignored. This is due to the fact that permission sets are used to conceal multiple app specific permission under a single item in the consent screen. Because of this only resources (record types and lexicon methods) under the authority of the Lexicon document author can be listed in permission sets.
```